frontend-react/src/styles/_uswds_overrides.scss

```diff
@@ -110,6 +112,27 @@ main {
   overflow-x: auto;
 }
 
+// Message Tracker specific styles for Accordion component
+.message-tracker {
```

Comment:
Should this class live here or be moved to another scss file?

Reply:
Are these changes across all `<Accordion />` components across the sites? If so, we should remove the `.message-tracker` scoping to the styles. Otherwise, I'd recommend we make either a `MessageTracker.module.scss` file or we can add to `_components.scss` file and we'll add component-scoped CSS in Stephen's rewrite.

Reply:
Will move.